text/0000-add-ember-page-title-addon.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2020-06-24
+- Relevant Team(s): Ember CLI
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+- Authors: Benjamin Jegard, Melanie Sumner, Ricardo Mendes
+
+# Add ember-page-title to the app blueprint
+
+## Summary
+
+Add [`ember-page-title`](https://github.com/adopted-ember-addons/ember-page-title) to the default blueprint for new Ember apps as a way to provide improved out-of-the-box (OOB) accessibility for Ember applications.
```

Comment:
In the examples, I see: 

![image](https://user-images.githubusercontent.com/199018/86475517-d5aa7300-bd12-11ea-9d06-d9587eb39d68.png)

Is there a way to omit the `Segment > Segment` stuff? I wouldn't want a breadcrumb in the title all the time :D

Reply:
The separator can be specified for each use of `{{page-title}}` and you can also set the default separator in `config/environment.js` (everything is explained in the [README](https://github.com/adopted-ember-addons/ember-page-title#page-title-helper))

Reply:
it's not just the separator, I feel like there is duplication.

Like, I'd want `PostTitle | Posts | Ember.js`, not `Post Title > Posts | Ember.js > Blog > etc`

Reply:
I think there's an issue on the doc page. All examples affect each others ><, for examples the part `Tim Evans (@ timmyce)` comes from the last example ><.

Reply:
Documentation page has been improved lately. This should no longer be an issue.